consoles/sshVirtsh.pm

```diff
@@ -412,6 +412,10 @@ sub add_disk {
             }
             else {
                 $self->run_cmd(sprintf("rsync -av '$args->{file}' '$basedir/%s'", $file_basename)) && die 'rsync failed';
+                if ($file_basename =~ /(.*)\.xz/) {
+                    $self->run_cmd(sprintf("nice ionice unxz -f -k '$basedir/%s'", $file_basename)) unless (-e $basedir . $1);
```

Comment:
who/what will delete this later?

Reply:
The image gets uncompressed inside /var/lib/libvirt/images so I don't think I have to remove it from there since it's done through a service on the host. If this is not what you asked I am not sure I understood the question.